.gitignore

```
@@ -0,0 +1,13 @@
+.DS_Store
+.idea
+
+.ddev/*
+!.ddev/config.yaml
+!.ddev/docker-compose.environment.yaml
+
+/public/*
+!/public/typo3conf
+/public/typo3conf/*
```

```suggestion
/public/typo3conf/*
!/public/typo3conf/LocalConfiguration.php
!/public/typo3conf/AdditionalConfiguration.php
```
Without the LocalConfiguration.php the Database-Dump is not usable.